.github/workflows/docker-nightly-publish.yml

```diff
@@ -111,6 +116,34 @@ jobs:
           docker tag deepjavalibrary/djl-serving:${{ env.SERVING_VERSION }} deepjavalibrary/djl-serving:latest
           docker push deepjavalibrary/djl-serving:latest
 
+  run-integration-tests:
+    if: ${{ inputs.mode == 'nightly' && !inputs.skip_integ_test }}
+    needs: [nightly-build, nightly-aarch64]
+    uses: ./.github/workflows/integration.yml
+    secrets: inherit
+    with:
+      djl-version: temp
+
+  push-to-dockerhub:
+    runs-on: ubuntu-latest
+    needs: [run-integration-tests]
+    strategy:
+      matrix:
+        arch: [ cpu, cpu-full, pytorch-inf2, pytorch-gpu, tensorrt-llm, lmi ]
+    steps:
```

Comment:
It looks like we're a binary success/fail situation here for all containers. Integ tests for all containers must pass for any new nightly publish. We should think through what it takes to make this more granular, which may involve splitting up the current integration.yml file into a few separate worfklows, or adding specific steps here that run tests on a container by container basis